packages/cms-cli/commands/init.js

```diff
@@ -0,0 +1,83 @@
+const { version } = require('../package.json');
+const inquirer = require('inquirer');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const { logger } = require('@hubspot/cms-lib/logger');
+
+const {
+  addConfigOptions,
+  addLoggerOptions,
+  setLogLevel,
+} = require('../lib/commonOpts');
+const { logDebugInfo } = require('../lib/debugInfo');
+const {
+  trackCommandUsage,
+  addHelpUsageTracking,
+} = require('../lib/usageTracking');
+const { PORTAL_API_KEY, PORTAL_ID, PORTAL_NAME } = require('../lib/prompts');
+
+const COMMAND_NAME = 'init';
+const HUBSPOT_CONFIG_YAML_FILE_NAME = 'hubspot.config.yaml';
+
+const promptUser = async promptConfig => {
+  const prompt = inquirer.createPromptModule();
+  return prompt(promptConfig);
+};
+
+const generateHubSpotConfigYAML = configData => {
+  logger.log('Generating hubspot.config.yaml');
+  try {
+    fs.writeFileSync(HUBSPOT_CONFIG_YAML_FILE_NAME, yaml.safeDump(configData));
+    logger.log('Successfully generated hubspot.config.yaml');
+  } catch (e) {
+    logger.error(e);
```

Comment:
We have some handlers to help with printing errors.  Here you could use [`logFileSystemErrorInstance`](https://github.com/HubSpot/hubspot-cms-tools/blob/7a3e83130b4427b36bf2aaffc512615eedf4fcaa/packages/cms-lib/errorHandlers.js#L266). For [example](https://github.com/HubSpot/hubspot-cms-tools/blob/d594ead3a24bbbef3ee9e3afea68dfb4882a7404/packages/cms-lib/lib/config.js#L33).